src/test/java/org/opensearch/sdk/TestNetty4Transport.java

```diff
@@ -24,12 +24,12 @@
 
 public class TestNetty4Transport extends OpenSearchTestCase {
 
-    private ExtensionsRunner extensionsRunner;
     private ThreadPool threadPool;
+    private GetNetty4Transport getNetty4Transport = new GetNetty4Transport();
 
     @BeforeEach
     public void setUp() throws IOException {
-        this.extensionsRunner = new ExtensionsRunner();
+        new ExtensionsRunner();
```

Comment:
What are we doing here exactly?

Reply:
Before we start refactor `nettyTransport` method, we call the `nettyTransport` method from `ExtensionsRunner.java`. Since the `getNetty4Transport` moved to a new class, we do not need to call this method from `ExtensionsRunner.java`. So I use IDE delete this line, but the IDE do not help delete whole line, half line has been left. I will delete that in next commit.